features/buyer/award_flow_requirements.feature

```diff
@@ -0,0 +1,84 @@
+@awardflow @skip-staging
+Feature: Award Flow
+  In order to ensure the procurement process is fair and transparent
+  As a buyer within government
+  I want to be able to publish the result of my procurement process
+
+
+Background:
+  Given I am logged in as the buyer of a closed brief with responses
+
+  When I click the 'View your account' link
+  And I click the 'View your requirements' link
```

Comment:
Elsewhere we're duplicating tests for staging with the alternative buyer dashboard flow (with `@skip-local @skip-preview` etc) - it would be nice to have coverage for staging and be consistent with the other features. Is it possible to just duplicate the `Background` section with the different tags? (I have no idea if that would work!)